constraints.pro

```diff
@@ -267,13 +267,17 @@ gen_enforced_field(WorkspaceCwd, 'scripts.changelog:validate', ProperChangelogVa
   atomic_list_concat(['../../scripts/validate-changelog.sh ', WorkspacePackageName], ProperChangelogValidationScript).
 
 % All non-root packages must have the same "test" script.
-gen_enforced_field(WorkspaceCwd, 'scripts.test', 'jest') :-
+gen_enforced_field(WorkspaceCwd, 'scripts.test', 'jest --reporters=jest-silent-reporter') :-
```

Comment:
Copying over the changes to the root package script, what do you think about:

``` suggestion
gen_enforced_field(WorkspaceCwd, 'scripts.test', 'jest --collectCoverage=false --reporters=jest-silent-reporter') :-

Reply:
I think we might want to enable coverage reports and threshold checks for package-level, especially since CI runs the package-level scripts.